### DISCUSSION TITLE
Problem with Domain to app mapping

### DISCUSSION THREAD

**Discussion by Nisaba** [btcpayserver/btcpayserver#4719] [Technical Support]
Hello,
It seems that Domain to app mapping feature doesn't work :
When I access to the app via the redirection domain : 
The app is well displayed in the brower. I can fill the shopping cart. But if I click on the Confirmation Pay Button, the page is reloaded and nothing is done.
Whereas all works well when I access to the app via the direct btcpay url.

Thank you for your help.

**Comment by dennisreimann** [comment]
Does this happen with v1.7.12?

**Comment by Nisaba** [comment]
I run the BTCPay Server v1.7.7 version

**Reply by dennisreimann** [reply]
This will be fixed in v1.8 which we plan to release during the next days. Sorry for the inconvenience.
